compiler/crates/relay-config/src/project_config.rs

```diff
@@ -74,6 +75,10 @@ fn default_node_interface_id_field() -> StringKey {
     "id".intern()
 }
 
+fn default_connection_interface() -> ConnectionInterface {
+    ConnectionInterface::default()
+}
+
```

Comment:
Needed to add this default to avoid always having to supply `connectionInterface` in the config file.